crown-court-contribution/build.gradle

```diff
@@ -49,9 +49,11 @@ dependencies {
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-test'
     implementation group: 'org.springframework.security', name: 'spring-security-oauth2-jose'
     implementation group: 'org.springframework', name: 'spring-jms'
-	compileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombokVersion
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
Please replace Junit 4 dependancy below with Junit 5/Jupiter, I think it should be something like `org.junit.jupiter:junit-jupiter`

`    testImplementation 'junit:junit:4.13.1'
`

Reply:
It address already.